resources/views/themes/bimbala/layouts/app.blade.php

```diff
@@ -81,7 +81,6 @@
             <div class="max-w-7xl mx-auto py-16 px-4 sm:py-20 sm:px-6 lg:px-8 lg:flex lg:justify-between">
                 <div class="max-w-xl">
                     <h2 class="text-4xl font-extrabold text-white sm:text-5xl sm:tracking-tight lg:text-6xl">{{ $seo['seo_title']}}</h2>
-                    <p class="mt-5 text-xl text-gray-300">{{ $seo['seo_description'] }}</p>
```

Comment:
Better nest it in a conditional which checks if the `seo_title` and `seo_description` are the same. I've spent hours composing meaningful texts for that.

Reply:
Hopefully, this way is better.